docs/guides/connecting_to_your_data/how_to_choose_between_working_with_a_single_or_multiple_batches_of_data.md

```diff
@@ -0,0 +1,108 @@
+
+<Prerequisites>
+
+- [Understand the basics of Datasources in the V3 (Batch Request) API](../../reference/datasources.md)
+- [Understand the basics of requesting data with a Batch Requests](./how_to_get_a_batch_of_data_from_a_configured_datasource.md)
+
+</Prerequisites>
+
+## Steps
+
+The steps of this guide will allow you to quickly determine if you will want to use a single Batch or multiple Batches when creating Expectations and performing Validations.  This knowledge will then inform how you configure Data Assets in your Datasources.
+
+### 1. Determine if a single Batch or multiple Batches of data will be most beneficial in your future use cases
+
+#### When to work with a single Batch of data:
+
+In some cases, a single Batch of data is all you need (or all that is supported).  In particular:
+- When you are Validating data, you will need to specify a single Batch of data.  If you provide multiple Batches in a Batch Request, Validations will default to operating on the last Batch in the list.
+- The Runtime Data Connector does not support handling multiple Batches of data.  You will always configure your Data Assets to correspond to at most a single Batch of Data when you are using a Runtime Data Connector.
+- If you are creating Expectations that don't rely on per-Batch statistical analysis, or you want to quickly create some Expectations as part of an early Expectation Suite or exploratory data analysis, a single Batch is likely to be all that you need.  
+
+:::note 
+
+When Validating data you may include multiple *Batch Requests* in a single Checkpoint.  But only one Batch out of each Batch Request will be used in the Validation process.
+
+:::
+
+#### When to work with multiple Batches of data:
+
+If you are creating Expectations, you will benefit from ensuring your Data Assets and Batch Requests can return multiple Batches when:
+- You will be using a Data Assistant to generate an Expectation Suite that is populated with Expectations for you.
+- You will be using auto-initializing Expectations to automatically generate the parameters for Expectations that you individually define and add to an Expectation Suite.
+
+This is particularly true if you are working with Expectations that involve statistical analysis on a per-Batch basis.
+
+For example, let's say you are working with the NYC taxi data and intended to define an Expectation to validate that the median trip distance of rides taken within a given month falls in a specific range. You would first need to ensure you could divide your data up into months for analysis, which you could do by configuring your Datasource to define an individual Data Asset for each month.  Then, you *could* analyze each month individually to determine what the appropriate minimum and maximum values for your Expectation would be.  But that would be a lot of work that Great Expectations can automate for you.
```

Comment:
I feel like this is a critical point too. Do you think it would be strengthened with a more specific description? 

* What range does the median `fare` fall between for June and December? Next, can we write an Expectation that alters us if the value falls out of that range?